package.json

```diff
@@ -257,6 +257,11 @@
           "default": false,
           "description": "Defines if sorting is disable during organize imports."
         },
+        "typescriptHero.resolver.disableUnsedImportsRemove": {
```

Comment:
Could you rename the setting to: `disableImportRemovalOnOrganize` ?
or even `resolver.organize.removeImports` and default `true` ?